src/content/structured/components/hero/code.mdx

```diff
@@ -19,35 +19,27 @@ tabs:
   ]
 ---
 
-import { IcHero, IcButton } from "@ukic/react";
+import { IcHero, IcButton, IcSearchBar, IcCard } from "@ukic/react";
 
 ## Component demo
 
 export const snippets = [
   {
     language: "Web component",
     snippet: `<ic-hero
-    heading="New coffee launches 14 September 2022"
-    subheading="Brand new flavours now available! Irresistible. Caramel. Decadence. Sugary.">
-  <ic-button variant="primary" appearance="light" slot="interaction">
-  Order now
-  </ic-button>
-  <ic-button variant="secondary" appearance="light" slot="interaction">
-  Submit new flavour
-  </ic-button>
+  heading="New coffee launches 14 September 2022"
+  subheading="Brand new flavours now available! Irresistible. Caramel. Decadence. Sugary.">
+  <ic-button variant="primary" appearance="light" slot="interaction">Order now</ic-button>
```

Comment:
In this file, here's some inconsistency with having 'appearance="light"' on the buttons in the code snippets e.g. in the React 'Card right content' one, the prop isn't there. 

I don't think we need the prop - in the previews when it isn't used (e.g. line 226) they appear correctly. I think the light / dark appearance is handled with the ic-theme set-up we have now - please could you remove the 'appearance="light"' from the snippets?